src/components/History/History.js

```diff
@@ -0,0 +1,43 @@
+import { useVideo } from "../../contexts/video-context";
+import "./history.css";
```

Comment:
As far as I know, you should import your custom CSS at the very bottom, to avoid any styles getting overridden.